kgforge/specializations/resolvers/ontology_resolver.py

```diff
@@ -57,33 +60,35 @@ def _resolve(self, text: Union[str, List[str]], target: Optional[str], type: Opt
             isDefinedBy ?isDefinedBy ;
             notation ?notation
         }} WHERE {{
-            ?id a ?type ;
-                label ?label ; 
-            OPTIONAL {{
-            ?id subClassOf ?subClassOf ;
-            }}
-            OPTIONAL {{
-            ?id definition ?definition ;
-            }}
-            OPTIONAL {{
-            ?id prefLabel ?prefLabel .
-            }}
-            OPTIONAL {{
-            ?id altLabel ?altLabel .
-            }}
-            OPTIONAL {{
-            ?id isDefinedBy ?isDefinedBy .
-            }}     
-            OPTIONAL {{
-            ?id notation ?notation .
-            }}    
-            {{
-            SELECT * WHERE {{
-                {{ {0} ; label ?label {1} }} UNION
-                {{ {0} ; notation ?notation {2} }} UNION
-                {{ {0} ; prefLabel ?prefLabel {3} }} UNION
-                {{ {0} ; altLabel ?altLabel {4} }}
-            }} LIMIT {5}
+            GRAPH ?g {{
```

Comment:
I would add Graph clause in the [agent resolver](https://github.com/BlueBrain/nexus-forge/blob/f94700222e78968e00849aa07d7e978f5109bd89/kgforge/specializations/resolvers/agent_resolver.py#L53) as well

Reply:
done